test/dockerpy/images.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import docker
+
+busybox = "docker.io/library/busybox:latest"
+
+def get_client():
+   return docker.DockerClient(base_url="unix://var/run/docker.sock")
+
+class TestImages(unittest.TestCase):
+    client = ""
+    def test_get_image(self):
+        client = get_client()
+        bb = client.images.get("busybox")
+        self.assertTrue(busybox in bb.tags)
```

Comment:
```suggestion
        self.assertTrue(busybox in bb.RepoTags)
```
From https://docker-py.readthedocs.io/en/1.6.0/api/#images